### DISCUSSION TITLE
After integrating tailwindcss according to the document, postcss does not take effect

### DISCUSSION THREAD

**Discussion by ghost** [web-infra-dev/rsbuild#4295] [General]
版本信息: 
"vue": "^3.5.13",
"tailwindcss": "^3.4.17",
"@rsbuild/core": "^1.1.13",
"@rsbuild/plugin-sass": "^1.1.2",
"@rsbuild/plugin-vue": "^1.0.5"

tailwind.config.mjs文件:
`/** @type {import('tailwindcss').Config} */
export default {
  content: ['./src/**/*.{js,vue}', './index.html'],
  theme: {
    extend: {}
  },
  plugins: []
}
`
postcss.config.mjs文件：
`export default {
  plugins: {
    tailwindcss: {}
  }
}`

tailwind.css文件：
@tailwind base;
@tailwind components;
@tailwind utilities;

在main.js中引入:
import '@styles/tailwind.css'

项目启动后，tailwindcss能正常使用，但是没有给css属性添加前缀，如下:
![image](https://github.com/user-attachments/assets/d97062ac-09f0-438c-bbf6-7f8c89922b92)

期望给display: flex属性加前缀

**Comment by chenjiahan** [comment]
Flex prefixes are not required in modern browsers. For legacy browsers, you can configure browserslist, see https://rsbuild.dev/guide/advanced/browserslist

**Reply by ghost** [reply]
got it
